moj-node/server/views/pages/games/chess.html

```diff
@@ -87,7 +88,7 @@ <h1 id="title" class="govuk-heading-xl govuk-!-margin-bottom-4">Chess</h1>
             <input type="hidden"  id="timeInc" value="2"/>
             {# </div> #}
 
-            <div class="govuk-form-group">
+            {#<div class="govuk-form-group">
```

Comment:
Why we commenting this out today?

Reply:
I'll put it back but it needs discussing, I think unless you really understand your chess then it's just confusing. I'd never heard of it until I saw this game 😄

Reply:
...and we are supposed to be making it easier ;)